labs/database/connect.py

```diff
@@ -15,18 +15,20 @@
 def db_connector():
     def decorator(original_function):
         def new_function(*args, **kwargs):
+            if settings.TEST_ENVIRONMENT:
```

Comment:
why do we need a specific connection for test environment?

Reply:
Added a comment to make it clearer why it's needed.